tasks/image-orientation/results.csv

```diff
@@ -0,0 +1,6 @@
+SampleID,TrueOrientation(degrees),PredictedOrientation(degrees),TrueSin,PredictedSin,TrueCos,PredictedCos

```

Comment:
should results.csv file be part of the template?

Reply:
No.